app/gui/view/src/project.rs

```diff
@@ -26,6 +27,45 @@ use ensogl_hardcoded_theme::Theme;
 
 
 
+// ============================
+// === WayOfOpeningSearcher ===
+// ============================
+
+/// An enum describing how the searcher was opened.
+#[derive(Clone, CloneRef, Copy, Debug, PartialEq)]
+pub enum WayOfOpeningSearcher {
+    /// New node was created by opening the searcher or the node is being edited.
+    NodeEdited(NodeId),
```

Comment:
Node Edited sounds like the node was edited. I would never understand that as node being edited. So this name is misleading,

Reply:
Would `NodeEditing` be better?